recipes/fish2eod/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "fish2eod" %}
+{% set version = "1.0a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/aaronshifman/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5ff1a9d5e5e8f10871ced326d4b86f9ecd0616e760e6beb983dfbebcae26f6c2
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
```

I see this package is pure Python (though it only advertises support for LInux/MacOS). Any particular need for the skip on Windows? If it's due to missing dependencies, we can make this a noarch python package instead by:
```suggestion
  noarch: python
```

---

Thank you @dopplershift for the comments. I've taken your suggestions for the other comments. The reason I didn't go with the noarch build was because there are datafiles (specified in MANIFEST.IN) that are needed at runtime. There was an issue with the naming in the packaging process and the required files weren't being found. Having separate builds seemed to resolve the issue

---

Would you be willing to switch this over to noarch at least temporarily in this build so I can see the problem (there are no windows failures in the CI logs I can see on this PR)? Having a pure Python package as noarch is preferable because it gets us down to a single build rather than the 20-30 builds that you get with all the python version and architecture permutations we support.

---

@dopplershift I must be mis-remembering from a previous recipe experiment because the macos / linux builds ran fine, the windows build failed as expected because the dependencies don't exist for windows. So I'm fine with `noarch`